reactive/kubernetes_worker.py

```diff
@@ -242,6 +242,49 @@ def shutdown():
     service_stop('snap.kube-proxy.daemon')
 
 
+@when_not('config.changed.open-ports')
```

Comment:
This seems an odd thing to do. Why do we want to run this when the config hasn't changed?

Reply:
typo. Should've been @when, thanks a lot for that :)